be/src/olap/olap_common.h

```diff
@@ -416,7 +418,8 @@ struct RowsetId {
             auto [_, ec] = std::from_chars(rowset_id_str.data(),
                                            rowset_id_str.data() + rowset_id_str.length(), high);
             if (ec != std::errc {}) [[unlikely]] {
-                LOG(FATAL) << "failed to init rowset id: " << rowset_id_str;
+                LOG(WARNING) << "failed to init rowset id: " << rowset_id_str;
+                high = unique_rowset_id_next_high();
```

Comment:
why not just
```
RowsetId rsid = next_rowset_id();
init(1, rsid.high, 0, 0);
```
No need to introduce an extra function `UniqueRowsetIdGenerator::next_high()`